release-team/role-handbooks/branch-manager/README.md

```diff
@@ -181,3 +182,16 @@ The current documentation in the [contributor guide for cherry picks](https://gi
 The cherry pick script is also fairly self documenting in terms of how to invoke the command.
 
 There has been quite a bit of recent discussion (see: [1](https://github.com/kubernetes/community/pull/2408), [2](https://github.com/kubernetes/community/pull/1980)) around improving both the cherry pick process process and its documentation.  Watch for and contribute to improvements.
+
+# Staging Repositories
+
+The [publishing-bot](https://github.com/kubernetes/publishing-bot) is responsible for publishing the code in
+[staging](https://git.k8s.io/kubernetes/staging/src/k8s.io) to their own repositories.
+
+The bot also syncs the Kubernetes version tags to the published repos, prefixed with `kubernetes-`.
+For example, if you check out the `kubernetes-1.13.0` tag in client-go, the code you get is exactly the same
+as if you check out the `v1.13.0` tag in Kubernetes, and change the directory to `staging/src/k8s.io/client-go`.
+
+[client-go](https://github.com/kubernetes/client-go) follows [semver](http://semver.org/)
+and has its own release process. This release process and the publishing-bot are maintained
+by SIG API Machinery. In case of any questions related to the published staging repos, please ask this SIG.
```

Comment:
The client-go release process is pretty much me and @sttts.

But I'm not sure if we do this as a part of SIG API Machinery or SIG Release? :)

Reply:
:) IMHO, it doesnt hurt to have this info here and link to SIG API Machinery page from here. I will also let @dougm chime here with his preference.

Reply:
>  link to SIG API Machinery page from here.

I linked to the OWNERS file for publishing-bot because that is more reflective of people who should be reached out to in case of problems.